graylog2-web-interface/src/views/components/dashboard/BigDisplayModeHeader.jsx

```diff
@@ -36,8 +36,14 @@ const BigDisplayModeHeader = ({ view: { activeQuery, view } = {} }: Props) => {
 
 BigDisplayModeHeader.propTypes = {
   view: PropTypes.shape({
-    title: PropTypes.string.isRequired,
-  }).isRequired,
+    view: PropTypes.shape({
+      title: PropTypes.string.isRequired,
+    }),
+  }),
+};
+
+BigDisplayModeHeader.defaultProps = {
+  view: undefined,
 };
```

Comment:
This throwed a PropType error. I removed the `isRequired` and added `undefined` as a default props, because it's the state, before the store got connected.